flow/layers/image_filter_layer_unittests.cc

```diff
@@ -97,18 +97,23 @@ TEST_F(ImageFilterLayerTest, SimpleFilter) {
   EXPECT_TRUE(layer->needs_painting(paint_context()));
   EXPECT_EQ(mock_layer->parent_matrix(), initial_transform);
 
-  SkPaint filter_paint;
-  filter_paint.setImageFilter(layer_filter);
-  layer->Paint(paint_context());
-  EXPECT_EQ(mock_canvas().draw_calls(),
-            std::vector({
-                MockCanvas::DrawCall{
-                    0, MockCanvas::SaveLayerData{child_bounds, filter_paint,
-                                                 nullptr, 1}},
-                MockCanvas::DrawCall{
-                    1, MockCanvas::DrawPathData{child_path, child_paint}},
-                MockCanvas::DrawCall{1, MockCanvas::RestoreData{0}},
-            }));
+  DisplayListBuilder expected_builder;
```

Comment:
Ooh, thank you for doing these conversions. I was giving another engineer a pass on updating these, but these conversions are going to eventually be needed when we get closer to a day when we can remove Skia-isms entirely from the code base.